lola.html

```diff
@@ -417,6 +416,7 @@ <h3>Bootstrapping URL discovery</h3>
     "name": "Evan Prodromou",
     "inbox": "https://example.com/evanp/inbox",
     "outbox": "https://example.com/evanp/outbox",
+    "content": "https://example.com/evanp/content",
```

Comment:
I'm not sure this collection is really defined anywhere-- I think mostly FEPs use the term "content" as a general term for outbox contents, i.e. Activities, Static Objects (i.e. Notes, Posts, Media envelopes), Attachments, and extension-defined objects?